apps/web/src/app/app/(dashboard)/[workspaceSlug]/status-pages/page.tsx

```diff
@@ -30,7 +28,14 @@ export default async function Page({
     workspaceSlug: params.workspaceSlug,
   });
 
-  const isLimit = (pages?.length || 0) >= limit;
+  const workspace = await api.workspace.getWorkspace.query({
+    slug: params.workspaceSlug,
+  });
+
+  const isLimit =
+    (monitors?.length || 0) >=
+    allPlans[workspace?.plan || "free"].limits["status-pages"];
+
```

Comment:
Another idea, is to extract into a helper function inside of `@openstatus/plans`.

```ts
export function isLimitReached(workspaceSlug: string, feature: "monitors" || "status-pages"): boolean {
  // ...
}
```